### PR TITLE
[webhooks] add v1 of a guide

### PR DESCRIPTION
**Pull request by gedankenstuecke**
As suggested in #1801, I've gone and made a first version of how to install and setup `webhooks` on Uberspace. 

So far I didn't go into detail regarding using this for CI/CD with Codeberg's Woodpecker-based CI, but I'd be happy to either add that in here, or expand the [existing CI guide](https://lab.uberspace.de/howto_automatic-deployment/), depending on what folks think is better!

### CODE REVIEW

**Review by brutus — COMMENTED** [review]
Hi. Thanks for the PR! LGTM, here are some suggested edits though.

File: source/guide_webhook.rst [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+.. code-block::
+
+  - id: test-hook
+    execute-command: /home/isabell/hook-triggered.sh
+    command-working-directory: /home/isabell
+    response-message: the test-hook was triggered!
+
+
+The :code:`id` will be the endpoint at which the webhook will be listening, while :code:`execute-command` specifies which command will be run when the hook is triggered, for testing this can just be a simple print-statement, e.g. you could have your :code:`hook-triggered.sh` be as simple as the below to get started.
+
```
[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply source/guide_webhook.rst:88]: Great point, added that in!

File: source/guide_webhook.rst [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+.. code-block:: ini
+
+    [program:webhook]
+    directory=%(ENV_HOME)s/webhook
+    command=/home/isabell/webhook/webhook -hooks hooks.yaml -logfile webhook.log -hotreload
+    startsecs=15
+
+The :code:`-hotreload` parameter will automatically reload your hook-definitions whenever you edit them. This config will also write a logfile with the outputs in :code:`~/webhook/webhook.log`.
+
```
[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply source/guide_webhook.rst:113]: Oh yes, very good point. I had kept that from my test-setup, but you're right in that it's the better and expected way!

File: source/guide_webhook.rst [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+Configuration
+=============
+
+With webhook installed and a first hook written, we now need to setup a service for `webhook` to run automatically.
+
+Creating a service
+------------------
+
+We can create a file ``~/etc/services.webhook.ini`` with the following content (adapt the values to your own directories if you have adapted them!):
```
[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply source/guide_webhook.rst:103]: thanks for catching this!

File: source/guide_webhook.rst [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+
+Testing your first webhook
+==========================
+
+By now, webhook is running and waiting to be triggered. In this example, your test-hook would be listening at :code:`https://webhook.isabell.uber.space/hooks/test-hook`. You can try using curl from your host machine (or your uber space):
+
+::
+
+ [your@host ~]$ curl https://webhook.isabell.uber.space/hooks/deploy-api
```
[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply source/guide_webhook.rst:145]: Oh yes, I must have missed this one replace when copying from my actual setup!

**Comment by gedankenstuecke** [comment]
Thanks so much @brutus, those were all really useful! I've pushed an update to hopefully fix those :)

**Comment by gedankenstuecke** [comment]
Actually noticed I forgot a couple of non-adapted log-paths, but now should be ready for another review 🙂

**Review by brutus — APPROVED** [review]

**Pull request merged**